README.md

```diff
@@ -42,3 +42,6 @@ Quickstart guide:
 
 6. Point your browser to ```http://127.0.0.1:8080```
 (or whatever you set host/port to in step 5)
+
+## To deploy this repo to a docker
```

Comment:
This is the how-to for deploying the changes to a specific machine and updating the example site, yeah? If so, might make more sense to title this `To deploy your changes to a particular environment`

Unless this is only part of the work in getting it onto an actual box, cuz I just noticed that this literally only builds the docker container. How do we get it onto an actual box?